Command/RouterDebugExposedCommand.php

```diff
@@ -23,18 +26,23 @@
  *
  * @author      William DURAND <william.durand1@gmail.com>
  */
-class RouterDebugExposedCommand extends RouterDebugCommand
+class RouterDebugExposedCommand extends ContainerAwareCommand
```

Comment:
This command has to be defined as a service, otherwise it won't be registered in Symfony 4.

And once defining it as a service, we can use DI in it instead of making it ContainerAware, allowing to avoid making the extractor service public (it is public only because of this command being ContainerAware)

Reply:
and commands as services were supported in 2.7 already https://symfony.com/doc/2.7/console/commands_as_services.html